packages/react/src/patterns/sub-patterns/TableOfContents/TableOfContents.js

```diff
@@ -33,34 +31,31 @@ const { prefix } = settings;
  */
 const TableOfContents = ({ menuItems, children, menuLabel, theme }) => {
   const [selectedId, setSelectedId] = useState(menuItems[0].id);
-  const [selectedTitle, setSelectedTitle] = useState(menuItems[0].title);
-
+  const [selectedTitle, setSelectedTitle] = useState(menuItems[0].name);
+  const useMenuItems = menuItems ? [...menuItems] : findMenuItems();
```

Comment:
You may want to set `useMenuItems` as a let, but define it inside the `useEffect`. If you try to load `TableOfContents` on the server side, the method `_findMenuItems` would fail.